src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationReferenceBuilder.java

```diff
@@ -13,103 +13,103 @@ private NotificationReferenceBuilder() {
             // do not instantiate
         }
 
-        public static String referenceForClaimant(final String claimReferenceNumber) {
+        public static String referenceForClaimant(String claimReferenceNumber) {
             return reference(TEMPLATE, CLAIMANT, claimReferenceNumber);
         }
 
-        public static String referenceForDefendant(final String claimReferenceNumber) {
+        public static String referenceForDefendant(String claimReferenceNumber) {
             return reference(TEMPLATE, DEFENDANT, claimReferenceNumber);
         }
     }
 
     public static class CCJRequested {
 
-        public static final String TEMPLATE = "%s-ccj-requested-notification-%s";
+        public static String TEMPLATE = "%s-ccj-requested-notification-%s";
 
         private CCJRequested() {
             // do not instantiate
         }
 
-        public static String referenceForClaimant(final String claimReferenceNumber) {
+        public static String referenceForClaimant(String claimReferenceNumber) {
             return reference(TEMPLATE, CLAIMANT, claimReferenceNumber);
         }
     }
 
     public static class OfferMade {
 
-        public static final String TEMPLATE = "%s-offer-made-notification-%s";
+        public static String TEMPLATE = "%s-offer-made-notification-%s";
 
         private OfferMade() {
             // do not instantiate
         }
 
-        public static String referenceForClaimant(final String claimReferenceNumber) {
+        public static String referenceForClaimant(String claimReferenceNumber) {
             return reference(TEMPLATE, CLAIMANT, claimReferenceNumber);
         }
 
-        public static String referenceForDefendant(final String claimReferenceNumber) {
+        public static String referenceForDefendant(String claimReferenceNumber) {
             return reference(TEMPLATE, DEFENDANT, claimReferenceNumber);
         }
     }
 
     public static class OfferAccepted {
 
-        public static final String TEMPLATE = "to-%s-offer-accepted-by-claimant-notification-%s";
+        public static String TEMPLATE = "to-%s-offer-accepted-by-claimant-notification-%s";
 
         private OfferAccepted() {
             // do not instantiate
         }
 
-        public static String referenceForClaimant(final String claimReferenceNumber) {
+        public static String referenceForClaimant(String claimReferenceNumber) {
             return reference(TEMPLATE, CLAIMANT, claimReferenceNumber);
         }
 
-        public static String referenceForDefendant(final String claimReferenceNumber) {
+        public static String referenceForDefendant(String claimReferenceNumber) {
             return reference(TEMPLATE, DEFENDANT, claimReferenceNumber);
         }
     }
 
     public static class OfferRejected {
 
-        public static final String TEMPLATE = "to-%s-offer-rejected-by-claimant-notification-%s";
+        public static String TEMPLATE = "to-%s-offer-rejected-by-claimant-notification-%s";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "TEMPLATE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static TEMPLATE" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make TEMPLATE a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)